src/FSharpLint.Core/Rules/Conventions/UsedUnderscorePrefixedElements.fs

```diff
@@ -0,0 +1,56 @@
+module FSharpLint.Rules.UsedUnderscorePrefixedElements
+
+open System
+open FSharpLint.Framework
```

Comment:
@realmarv nit, please add EOL between `open System` and the other opens

Reply:
Done